product_docs/docs/biganimal/release/getting_started/overview.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: "Getting started overview"
+navTitle: "Overview"
+description: A high level description of the tasks needed to get started with BigAnimal 
+---
+
```

Comment:
Add an intro statement for Getting Started and the two options in the bulleted list. Link both options to the individual topics.

bulleted list can be something like this:
 - Using BigAnimal's cloud account
 - Using your own cloud account